packages/core/src/forms/control-group/control-group.element.ts

```diff
@@ -181,13 +181,13 @@ export class CdsInternalControlGroup extends LitElement {
     setAttributes(this, ['role', 'group'], ['cds-control-group', '']);
   }
 
-  firstUpdated(props: Map<string, any>) {
+  firstUpdated(props: PropertyValues<this>) {
     super.firstUpdated(props);
     this.associateLabelAndGroup();
     this.setupResponsive();
   }
 
-  updated(props: Map<string, any>) {
+  updated(props: PropertyValues<any>) {
```

Comment:
A couple of these have `any` since it seems that the `PropertyValues` type is not picking up private props using `@state` at the moment.